src/cljs/rems/fields.cljs

```diff
@@ -111,7 +111,7 @@
         value (:field/value opts)
         previous-value (:field/previous-value opts)
         max-length (:field/max-length opts)
-        info-text (linkify (localized (:field/info-text opts)))
```

Comment:
so, how did linkify cause this problem?

Reply:
i mean, this looks like a now links in the info text will not be heighlighted

Reply:
I pushed the call to linkify down to line 144 to be able to check if the original text is blank or not. `(str/blank? (linkify foo))` unfortunately doesn't work since `linkify` returns a sequence of strings and not a single string.

Reply:
right, should we then try to approach it from a different angle?
like maybe have another version of the linkify function?

Reply:
i would really like to have the linkyfying functionality still, lets see where it is used

Reply:
The linkifying functionality is still there for info texts. The call has just moved from this `let` to the use site on line 144.